pymc3/distributions/transforms.py

```diff
@@ -237,6 +238,53 @@ def jacobian_det(self, x):
 upperbound = UpperBound
 
 
+class Ordered(ElemwiseTransform):
+    name = "ordered"
+
+    def backward(self, x):
```

Comment:
Are you sure this is not forward?? See
https://discourse.pymc.io/t/sampling-uniformly-in-a-triangular-support/765/12?u=junpenglao